src/Data/Nat/Divisibility.agda

```diff
@@ -25,24 +25,59 @@ open import Relation.Binary.Definitions
   using (Reflexive; Transitive; Antisymmetric; Decidable)
 import Relation.Binary.Reasoning.Preorder as PreorderReasoning
 open import Relation.Binary.PropositionalEquality.Core
-  using (_≡_; _≢_; refl; sym; trans; cong; cong₂; subst)
+  using (_≡_; _≢_; refl; sym; cong; subst)
 open import Relation.Binary.Reasoning.Syntax
-import Relation.Binary.PropositionalEquality.Properties as PropEq
+open import Relation.Binary.PropositionalEquality.Properties
+  using (isEquivalence; module ≡-Reasoning)
+
+private
+  variable d m n o p : ℕ
+
+
+------------------------------------------------------------------------
+-- Definition and derived properties
+
+open import Data.Nat.Divisibility.Core public hiding (*-pres-∣)
+
+quotient≢0 : (m∣n : m ∣ n) → .{{NonZero n}} → NonZero (quotient m∣n)
+quotient≢0 m∣n rewrite _∣_.equality m∣n = m*n≢0⇒m≢0 (quotient m∣n)
+
+equalityᵒ : (m∣n : m ∣ n) → n ≡ m * (quotient m∣n)
+equalityᵒ {m = m} m∣n rewrite _∣_.equality m∣n = *-comm (quotient m∣n) m
+
+quotient-∣ : (m∣n : m ∣ n) → (quotient m∣n) ∣ n
+quotient-∣ {m = m} m∣n = divides m (equalityᵒ m∣n)
+
+quotient>1 : (m∣n : m ∣ n) → m < n → 1 < quotient m∣n
+quotient>1 {m} {n} m∣n m<n = *-cancelˡ-< m 1 (quotient m∣n) $ begin-strict
+    m * 1        ≡⟨ *-identityʳ m ⟩
+    m            <⟨ m<n ⟩
+    n            ≡⟨ equalityᵒ m∣n ⟩
+    m * quotient m∣n ∎
+  where open ≤-Reasoning
+
+quotient-< : (m∣n : m ∣ n) → .{{NonTrivial m}} → .{{NonZero n}} → quotient m∣n < n
+quotient-< {m} {n} m∣n = begin-strict
+  quotient m∣n     <⟨ m<m*n (quotient m∣n) m (nonTrivial⇒n>1 m) ⟩
+  quotient m∣n * m ≡⟨ _∣_.equality m∣n ⟨
+  n                ∎
+  where open ≤-Reasoning; instance _ = quotient≢0 m∣n
 
 ------------------------------------------------------------------------
--- Definition
+-- Relating _/_ and quotient
 
-open import Data.Nat.Divisibility.Core public
+n/m≡quotient : (m∣n : m ∣ n) .{{_ : NonZero m}} → n / m ≡ quotient m∣n
+n/m≡quotient {m = m} (divides-refl q) = m*n/n≡m q m
```

Comment:
The more I think about it, the more I think this lemma belongs in `DivMod` or even `Divisibility.Core`, for then the [two lemmas in `DivMod`](https://github.com/agda/agda-stdlib/blob/9708ca8a6b528180d59025892aa8df941b9e3141/src/Data/Nat/DivMod.agda#L187-L191) follow almost immediately; indeed would do so if moreover `equalityᵒ` (however we end up naming this) went with them... what do you think?